### ISSUE TITLE
Label Conversion, Training and inference

### ISSUE THREAD

**Issue by MyVanitar** [isht7/pytorch-deeplab-resnet#9]
Hello,

1) The original VOC PNG labels are in RGB format, but it seems that we have to convert them to the single pallet 1D PNGs (0, 1, 2, ... 255). May I ask you what tool have you used to make this conversion?

2) in the training parameters you mentioned:
`--LISTpath=<str>            Input image number list file [default: data/list/train_aug.txt]`
should this contain training images names or training labels names or both?

3) is there any code to make some visual inference?

**Comment by isht7** [comment]
1. you can do so using in python using a dictionary to create a mapping between each RGB color and a label.  A similar, but opposite thing has been done in this [file](https://github.com/DrSleep/tensorflow-deeplab-resnet/blob/master/deeplab_resnet/utils.py) using the function named `decode_labels`. This function converts  single pallet 1D PNGs (0, 1, 2, ... 255) to RGB.  You must train on all 10582 images. The full train images are not available just at the VOC server, and must be augmented from [here](http://www.eecs.berkeley.edu/Research/Projects/CS/vision/grouping/semantic_contours/benchmark.tgz) as written in the deeplab-resnet paper.
2.  It is assumed that a training image and it's corresponding ground truth have the same file name, but are present in different folders. (In VOC both have the same name by default, but the file extensions are different.)
3. There is a flag in the evaluation code to visualize each output, and compare with gt image and input image. Run `evalpyt2.py -h` to see info about each flag.
